devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -32,6 +32,9 @@ metadata {
 
 		attribute "colorName", "string"
 
+		//Generic fingerprint
+		fingerprint profileId: "0104", deviceId: "010D", inClusters: "0006, 0008, 0300", outClusters: "0019" // deviceId 0x010D is "Extended Color Light"
```

Comment:
`, outClusters: "0019"` can be removed since that's OTA and isn't used by the DTH